caravel/templates/thumbnail_button.html

```diff
@@ -0,0 +1,6 @@
+<form method="get" style="display:inline-block">
```

Comment:
more inline styles. Maybe move `.thumbnail-button` out, and use child selectors?